components/Trip/index.js

```diff
@@ -12,8 +12,7 @@ export default function Trip({
     <li>
       <Image src={image} width={100} height={50} alt="" />
       <p>
-        name: {title} startDate: {startDate} endDate:
-        {endDate} city: {city} country: {country}
+        <h2>{title}</h2> {startDate} - {endDate} <br></br> {city}, {country}
       </p>
     </li>
```

Comment:
A `p` element is not supposed to include a `h2` element (https://caninclude.glitch.me/caninclude?child=h2&parent=p). I would suggest the following:

```jsx
 <li>
      <Image src={image} width={100} height={50} alt="" />
  
        <h2>{title}</h2>
      <p> {startDate} - {endDate} <br></br> 
             {city}, {country}
      </p>
 </li>
```